linux/devdoc/error_handling_requirements.md

```diff
@@ -0,0 +1,41 @@
+# error handling
+
+## Overview
+
+Linux implentaton of GetLastError and SetLastError 
+
+## Exposed API
+
+```c
+uint64_t last_error_code;
```

Comment:
I'd remove this from here and the other spec file.  technically you could decided to keep the last_error_code in a lot of different containers so you don't have to advertise that here.